crates/sui-protocol-config/src/lib.rs

```diff
@@ -3375,6 +3376,19 @@ impl ProtocolConfig {
                     cfg.tx_context_ids_created_cost_base = Some(30);
                     cfg.tx_context_replace_cost_base = Some(30);
                     cfg.gas_model_version = Some(10);
+
+                    if chain != Chain::Mainnet {
+                        // Enable execution time estimate mode for congestion control on testnet.
+                        cfg.feature_flags.per_object_congestion_control_mode =
+                            PerObjectCongestionControlMode::ExecutionTimeEstimate(
+                                ExecutionTimeEstimateParams {
+                                    target_utilization: 30,
```

Comment:
any reason for 30?

Reply:
No reason - it seemed a bit safer to start than the 50 we eventually want to end up at. feel free to propose another number and I'll accept :)